### PR TITLE
Vb refactoring any to all

### PR DESCRIPTION
**Pull request by jwooley**
Translate fix for Issue #370 to VB

### CODE REVIEW

File: src/VisualBasic/CodeCracker/Refactoring/ChangeAnyToAllCodeFixProvider.vb [hunk 1]
```diff
@@ -0,0 +1,139 @@
...
+        Public NotOverridable Overrides Function GetFixAllProvider() As FixAllProvider
+            Return WellKnownFixAllProviders.BatchFixer
+        End Function
+
+        Public Overrides Async Function RegisterCodeFixesAsync(context As CodeFixContext) As Task
+            Dim diag = context.Diagnostics.First
+            Dim message = If(diag.Id = DiagnosticId.ChangeAnyToAll.ToDiagnosticId, "Change Any to All", "Change All to Any")
+            context.RegisterCodeFix(CodeAction.Create(message, Function(c) ConvertAsync(context.Document, diag.Location, c)), diag)
+            Await Task.Yield
+        End Function
```
Comment (giggio) [inline src/VisualBasic/CodeCracker/Refactoring/ChangeAnyToAllCodeFixProvider.vb]: Why await `Task.Yield`? This has an impact on the execution. I suggest you remove the `Async` modifier from the method and return `Task.FromResult(0)` as on the original method.

File: test/VisualBasic/CodeCracker.Test/Refactoring/ChangeAnyToAllTests.vb [hunk 2]
```diff
@@ -0,0 +1,172 @@
...
+        '<InlineData("
+        '    Dim ints = {1, 2}
+        '    Dim query = ints.Any(Function(i) (i = 1) <> False)", "
+        '    Dim ints = {1, 2}
+        '    Dim query = Not ints.All(Function(i) (i = 1) = False)")>
+        '<InlineData("
+        '    Dim ints = {1, 2}
+        '    Dim query = ints.Any(Function(i) False <> (i = 1))", "
+        '    Dim ints = {1, 2}
+        '    Dim query = Not ints.All(Function(i) False = (i = 1))")>
```
Comment (giggio) [inline test/VisualBasic/CodeCracker.Test/Refactoring/ChangeAnyToAllTests.vb]: Why all this commented code? Shouldn't they be uncommented?

Comment (jwooley) [inline test/VisualBasic/CodeCracker.Test/Refactoring/ChangeAnyToAllTests.vb]: Sorry. Forgot to uncomment before checking in. Revising now. All tests still pass.

File: src/VisualBasic/CodeCracker/Refactoring/ChangeAnyToAllCodeFixProvider.vb (hunk 1 above)
Comment (jwooley) [inline src/VisualBasic/CodeCracker/Refactoring/ChangeAnyToAllCodeFixProvider.vb]: Fixed

**Comment by giggio** [comment]
@jwooley Cool! Tks! :)

On that task comment I had meant to replace the `Await` with Task.FromResult, without the `Await`. See 161fc10 for the quick fix I applied.

**Pull request merged**